### PR TITLE
[IOTDB-2304]Library-UDF Data Repairing Functions

### PR DESCRIPTION
**Pull request by trin1t**
## Description
Functions, tests and documents for data repairing.

### Function List
TimestampRepair
ValueFill
ValueRepair

<hr>

This PR has:
- [x] been self-reviewed.
    - [ ] concurrent read
    - [ ] concurrent write
    - [ ] concurrent read and write 
- [x] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. 
- [ ] added or updated version, __license__, or notice information
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious 
  for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold 
  for code coverage.
- [x] added integration tests.
- [ ] been tested in a test IoTDB cluster.

<hr>

### CODE REVIEW

**Comment by SteveYurongSu** [comment]
```pom
    <artifactId>library-udf</artifactId>
    <version>2.0.0</version>
```
The version is better to be the same as its parent's pom.

File: docs/zh/UserGuide/UDF-Library/Data-Repairing.md [hunk 1]
```diff
@@ -135,9 +135,9 @@ select timestamprepair(s1) from root.test.d2
 **备注：** AR 模型采用 AR(1)，时序列需满足自相关条件，否则将输出单个数据点 (0, 0.0).
 
 ### 使用示例
-
 #### 使用 linear 方法进行填补
 
+>>>>>>> master:docs/zh/UserGuide/UDF-Library/Data-Repairing.md
```
Comment (qiaojialin) [inline docs/zh/UserGuide/UDF-Library/Data-Repairing.md:140]: remove

File: library-udf/src/main/java/org/apache/iotdb/library/drepair/UDTFValueRepair.java [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    method = parameters.getStringOrDefault("method", "screen");
+    minSpeed = parameters.getDoubleOrDefault("minSpeed", Double.NaN);
+    maxSpeed = parameters.getDoubleOrDefault("maxSpeed", Double.NaN);
+    center = parameters.getDoubleOrDefault("center", 0);
+    sigma = parameters.getDoubleOrDefault("sigma", Double.NaN);
+  }
+
+  @Override
+  public void transform(RowWindow rowWindow, PointCollector collector) throws Exception {
+    // 设置修复方法及参数
```
Comment (qiaojialin) [inline library-udf/src/main/java/org/apache/iotdb/library/drepair/UDTFValueRepair.java:74]: use English

**Review by qiaojialin — CHANGES_REQUESTED** [review]

File: docs/zh/UserGuide/UDF-Library/Data-Repairing.md (hunk 1 above)
Comment (FrankHWD) [reply docs/zh/UserGuide/UDF-Library/Data-Repairing.md]: Done.

File: library-udf/src/main/java/org/apache/iotdb/library/drepair/UDTFValueRepair.java (hunk 2 above)
Comment (FrankHWD) [reply library-udf/src/main/java/org/apache/iotdb/library/drepair/UDTFValueRepair.java]: Done.

**Review by qiaojialin — APPROVED** [review]

**Pull request merged**